workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -78,7 +81,7 @@ def datetime_decode(obj):
 
 
 class Command(BaseCommand):
-    help = ''
+    help = 'Check if events have had updated their metadata.'
```

Comment:
'Check if events have had their metadata updated' - your grammar is *almost* right, but *shrug* English...